.gitignore

```diff
@@ -9,6 +9,7 @@ conf.pri
 Makefile*
 *.pyc
 *.log
+/winconf-user.pri
```

Comment:
Do slash required here?

Reply:
It does mean, that only file in the root will match, without forward slash it will match in every subdir.